package.json

```diff
@@ -46,7 +46,7 @@
     "@salesforce/core": "^2.33.1",
     "@salesforce/kit": "^1.5.17",
     "@salesforce/source-deploy-retrieve": "^5.9.4",
-    "isomorphic-git": "^1.9.2",
+    "isomorphic-git": "^1.14.0",
```

Comment:
the `iso-git` maintainers mention that `iso-git` doesn't strictly adhere to semver, and pinning is [recommended](https://github.com/isomorphic-git/isomorphic-git/pull/1507#issuecomment-1068935562 )